### PR TITLE
Jenkinsfile for batch test run sequentially using kubeconfig

### PR DESCRIPTION
**Pull request by aganesh-suse**
#### Proposed Changes ####

Jenkinsfile for batch test run sequentially using kubeconfig

#### Testing ####

Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.

2. Verify code lint; we should not have errors.

3. Update the documentation if needed.

4. Update makefile and docker run if adding new tests.

5. Run your tests at least 4 times with all configurations needed and possible.

6. If needed test with different os types.

### CODE REVIEW

File: scripts/Jenkinsfile_tests_via_kcfg [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                string(name: 'KUBECONFIG', value: "${KUBECONFIG}"),                                                           
+              ]
+
+              switch(test_name) {
+                case "validatecluster":
+                  try {
+                    build job: "${env.PRODUCT_NAME}_validate_cluster", parameters: install_params
+                  } catch(err) {
+                    echo "Validate Cluster Test Error: " + err
+                    currentBuild.result = 'SUCCESS'
```
Comment (mdrahman-suse) [inline scripts/Jenkinsfile_tests_via_kcfg:70]: Should we be setting this value as `SUCCESS`

File: scripts/Jenkinsfile_tests_via_kcfg [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                string(name: 'BRANCH', value: "${BRANCH}"),                                         
+                string(name: 'PRODUCT_NAME', value: "${PRODUCT_NAME}"),                    
+                string(name: 'INSTALL_VERSION', value: "${INSTALL_VERSION}"),
+                string(name: 'INSTALL_MODE', value: "${INSTALL_MODE}"),
+                string(name: 'INSTALL_CHANNEL', value: "${INSTALL_CHANNEL}"),              
+                string(name: 'HOSTNAME_PREFIX', value: "${HOSTNAME_PREFIX}"), 
+                string(name: 'TEST_DIRECTORY', value: test_name),
+                string(name: 'AWS_SSH_KEY_NAME', value: "${AWS_SSH_KEY_NAME}"),
+                string(name: 'AWS_SSH_PEM_KEY', value: "${AWS_SSH_PEM_KEY}"),
+                string(name: 'KUBECONFIG', value: "${KUBECONFIG}"),
```
Comment (mdrahman-suse) [inline scripts/Jenkinsfile_tests_via_kcfg:61]: Should probably be `KUBE_CONFIG`

Comment (aganesh-suse) [reply scripts/Jenkinsfile_tests_via_kcfg:61]: good catch :)

File: scripts/Jenkinsfile_tests_via_kcfg (hunk 1 above)
Comment (aganesh-suse) [reply scripts/Jenkinsfile_tests_via_kcfg:70]: if i catch and set it temporarily, we wont be able to proceed with the test. like a failure will stop us from running the rest of the tests. am thinking ll set some other var and check the finally to reset it maybe.

**Comment by mdrahman-suse** [comment]
Can we close this @aganesh-suse ?

**Pull request closed without merging**